hardware/Dummy.cpp

```diff
@@ -394,6 +396,9 @@ namespace http {
 			{
 				root["status"] = "OK";
 				root["title"] = "CreateVirtualSensor";
+				std::stringstream ss;
```

Comment:
Arg, after a test, the ID contains +82000, therefore, it is different from the virtual sensor id required during json requests afterwards. sstream could be removed, but I still can't use the ID in the sprintfed buffer as is.